src/Entity/Controller/SlotListBuilder.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Drupal\acquia_lift\Entity\Controller;
+
+use Drupal\acquia_lift\AcquiaLiftException;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Config\Entity\ConfigEntityListBuilder;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use GuzzleHttp\Exception\RequestException;
+
+/**
+ * Builds a listing of slot entities.
+ */
+class SlotListBuilder extends ConfigEntityListBuilder {
+
+  /**
+   * The Lift API Helper.
+   *
+   * @var \Acquia\LiftClient\Lift
+   */
+  protected $liftClient;
```

Comment:
Personally, I would consider SlotListBuilder not intend to be inherited by another class, so I would make all the non-public attributes and functions, such as $liftClient, private.